onthego/youtube.py

```diff
@@ -72,7 +72,10 @@ def download_to_tmp(self, track):
 
         tmp_path = get_tmp_path(best)
         print("    Downloading %s to %s" % (video_url, tmp_path))
-        best.download(tmp_path, quiet=True)
+        try:
+            best.download(tmp_path, quiet=True)
+        except IOError:
+            return None
```

Comment:
Here, my view was that the underlying library was failing on the download (the IOError seem to be reproduceable, I kept getting stuck at the same song) but I didn't dig on why it failed.
in a spotify-playlist usage, I don't want to have my soft crash and stop during runtime, maybe we could prepare an error log that could be saved with all the songs that failed to be downloaded.

Will see about changing the level of the error.